docker/install.sh

```diff
@@ -10,7 +10,7 @@ if [ -e "jenkins_env.sh" ]; then
 fi
 
 # Tests are run in jenkins which has a custom mysql instance just for testing
-sbt -J-Xms4g -J-Xmx4g test -Dmysql.host=172.17.42.1 -Dmysql.port=3310
+sbt -J-Xms4g -J-Xmx4g test -Dmysql.host=mysql -Dmysql.port=3306
```

Comment:
`mysql` is the name of the docker container.  because we're also running these tests through docker and the containers are linked, we don't have to use a static IP here anymore